tests/Integration/UseCases/CancelDonation/CancelDonationUseCaseTest.php

```diff
@@ -134,7 +144,8 @@ private function newUseCaseWithMailerMock( Donation $donation ): CancelDonationU
 
 		return new CancelDonationUseCase(
 			$this->getDonationRepositoryWithDonation( $donation ),
-			$mailer
+			$mailer,
+			new SucceedingAuthorizer()
```

Comment:
Shouldn't the authorizer failing also be a test case to cover all the code paths?